hnn_core/params_default.py

```diff
@@ -412,16 +409,7 @@ def _secs_L2Pyr():
         'basal_2': 2.72,
         'basal_3': 2.72
     }
-    sec_scales = {  # factor to scale the dipole by
-        'soma': 1.,
-        'apical_trunk': 1.,
-        'apical_oblique': 0.,
-        'apical_1': 1.,
-        'apical_tuft': 1.,
-        'basal_1': -1.,
-        'basal_2': -np.sqrt(2.) / 2.,
-        'basal_3': -np.sqrt(2.) / 2.
```

Comment:
If I recall correctly, the true morphology of these basal dendrites doesn't orient them at 45 degrees to the apical dendrite. How is the calculated dipole in this PR still matching the ground-truth dipole in our tests?

Reply:
It does orient at 45 degrees actually. Take a look at the points above:

```py
'apical_trunk': [[-50, 778, 0], [-50, 813, 0]],
'basal_3': [[-50, 715, 0], [56, 609, 0]],
```

apical -> 35j
basal_3 -> 106i + 106j

First vector is parallel to y axis and second one has equal components in x and y axis. Thus they are at 45 degrees.

Reply:
Oh okay, so were the axes in `plot_cell_morphology` not equivalent when we were reviewing this the other day?

Reply:
optical illusion I would say :-) Because the apical dendrite is longer than the basal, it looked as if the angle was smaller than it really was